nodes/src/nodes/tool_vectordb/IGlobal.py

```
@@ -0,0 +1,116 @@
+class IGlobal(IGlobalBase):
+    """Global state for tool_vectordb."""
+
+    driver: VectorDBDriver | None = None
+
+    def beginGlobal(self) -> None:
+        if self.IEndpoint.endpoint.openMode == OPEN_MODE.CONFIG:
+            return
+
+        cfg = Config.getNodeConfig(self.glb.logicalType, self.glb.connConfig)
+
+        server_name = str((cfg.get('serverName') or 'vectordb')).strip()
```

Ruff (0.15.7)

[warning] Avoid extraneous parentheses

Remove extraneous parentheses

(UP034)